charts/vela-core/crds/core.oam.dev_applications.yaml

```diff
@@ -860,6 +865,7 @@ spec:
                 required:
                 - finished
                 - mode
+                - status
```

Comment:
why the newly added fields is required?

Reply:
What about the compatiblity？

Reply:
Will make it a omit field. This field shows the status of workflow.